app/authentication/auth.py

```diff
@@ -61,7 +61,22 @@ def requires_admin_auth():
 
     if client == current_app.config.get('ADMIN_CLIENT_USER_NAME'):
         g.service_id = current_app.config.get('ADMIN_CLIENT_USER_NAME')
-        return handle_admin_key(auth_token, current_app.config.get('ADMIN_CLIENT_SECRET'))
+
+        if len(current_app.config.get('ADMIN_CLIENT_SECRETS')):
```

Comment:
don't think you need this if check. the for loop will just do nothing if the current app has an empty array

Reply:
Fixed